widgets/locate/FindAddress.js

```diff
@@ -185,7 +201,8 @@ define([
             var url = '//api.mapserv.utah.gov/api/v1/Geocode/{geocode.street}/{geocode.zone}';
 
             var options = {
-                apiKey: this.apiKey
+                apiKey: this.apiKey,
+                spatialReference: this.spatialReference
```

Comment:
Should add the new option to line 90-ish so that we know that it's an available parameter.